arduino/main/main.ino

```diff
@@ -23,185 +30,172 @@ bool OLD_KEY1_PRESENT = false;
 bool OLD_KEY2_PRESENT = false;
 
 // US needs 5V
-uint8_t const US_TRIG_PIN = A0;
-uint8_t const US_ECHO_PIN = A1;
+uint8_t const US_TRIG_PIN = 4;  // 34 not working
+uint8_t const US_ECHO_PIN = 5;  // 35 not working
 
-uint8_t const MOTOR1_PWM_PIN = 3;
-uint8_t const MOTOR1_IN1_PIN = 2;
-uint8_t const MOTOR1_IN2_PIN = 4;
+uint8_t const MOTOR1_PWM_PIN = 14; // EN
+uint8_t const MOTOR1_IN1_PIN = 26;
+uint8_t const MOTOR1_IN2_PIN = 27;
 
-uint8_t const MOTOR2_PWM_PIN = 6;
-uint8_t const MOTOR2_IN1_PIN = 5;
-uint8_t const MOTOR2_IN2_PIN = 7;
+uint8_t const MOTOR2_PWM_PIN = 25; // EN
+uint8_t const MOTOR2_IN1_PIN = 32;
+uint8_t const MOTOR2_IN2_PIN = 33;
 
 // RGB LEDs need 5V
-uint8_t const LED_PIN = A2;
+uint8_t const LED_PIN = 15;
 uint8_t const LED_COUNT = 2;
 
 // VOLTMETER
-uint8_t const VOLTAGE_PIN = A3;
+uint8_t const VOLTAGE_PIN = A6; // GPIO 34
 uint8_t const VOLTAGE_RATIO = 11;
 
-// I2C
-uint8_t const I2C_ADDRESS = 1;
+// BUZZER
+uint8_t const BUZZ_PIN = 13;
+
+double const ADC_MAX_VALUE = 4095.0;
+
+class FakeTagFinder
```

Comment:
This probably needs to go in its own file(s).

Reply:
Done